lib/config/config-ops.js

```diff
@@ -30,28 +30,28 @@ var RULE_SEVERITY_STRINGS = ["off", "warn", "error"],
 // Public Interface
 //------------------------------------------------------------------------------
 
-module.exports = {
+module.exports = class ConfigOps {
```

Comment:
I'm much more of the mindset that classes with static methods are better than objects when there is no internal state (which is to say, it's a namespace). Happy to defer to consensus, though, as this was really more of a forcing function to get the Babel setup working rather than anything else.